shared/src/acme/jws.rs

```diff
@@ -137,6 +169,18 @@ pub fn jws(
     })
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Identifier {
+    #[serde(rename = "type")]
+    pub type_: String,
```

Comment:
We're using `r#type` on other structs for Acme, so may as well continue
```suggestion
    pub r#type: String,
```